homeassistant/components/plex/server.py

```diff
@@ -98,7 +98,7 @@ def account(self):
         if not self._plex_account and self._use_plex_tv:
             try:
                 self._plex_account = plexapi.myplex.MyPlexAccount(token=self._token)
-            except Unauthorized:
+            except (BadRequest, Unauthorized):
```

Comment:
Bad Request is HTTP status code `400`, your initial post showed a 422. Does this work ?

Reply:
That's the `plexapi` exception name for fallback errors:
https://github.com/pkkid/python-plexapi/blob/37c857568d6c8c2e0f8dda396250872580a28810/plexapi/myplex.py#L186-L195